src/parquet/file/writer.cc

```diff
@@ -55,8 +55,8 @@ ParquetFileWriter::~ParquetFileWriter() {
 
 std::unique_ptr<ParquetFileWriter> ParquetFileWriter::Open(
     std::shared_ptr<OutputStream> sink, std::shared_ptr<GroupNode>& schema,
-    MemoryAllocator* allocator, const std::shared_ptr<WriterProperties>& properties) {
-  auto contents = FileSerializer::Open(sink, schema, allocator);
+    MemoryAllocator* allocator, const std::shared_ptr<WriterProperties>& props) {
```

Comment:
Ok. I've seen `props` in other places, e.g. https://github.com/apache/parquet-cpp/blob/4c7b5f2e46160969323bcdf3273b1d6b8ff32d53/src/parquet/file/reader-internal.h#L73 and https://github.com/apache/parquet-cpp/blob/4c7b5f2e46160969323bcdf3273b1d6b8ff32d53/src/parquet/file/reader-internal.h#L103 and tried to maintain consistent style.